tests/flex/test_pythonfunctioncontainer.py

```diff
@@ -61,3 +66,17 @@ def test_with_executor_wait(self):
             self.assertFalse(job.server.future.done())
             self.project.wait_for_job(job=job, interval_in_s=0.01, max_iterations=1000)
             self.assertTrue(job.server.future.done())
+
+    def test_with_internal_executor(self):
+        job = self.project.wrap_python_function(my_function_exe)
+        job.input["a_lst"] = [1, 2, 3, 4]
+        job.input["b_lst"] = [5, 6, 7, 8]
+        job.server.cores = 2
+        with self.assertRaises(ImportError):
+            job.executor_type = "Executor"
+        job.executor_type = ProcessPoolExecutor
+        job.executor_type = None
+        job.executor_type = "concurrent.futures.ProcessPoolExecutor"
```

Comment:
Better to extend these so to guarantee `._get_executor` then has the expected behaviour for each case

Reply:
Done